datalad/cmd.py

```diff
@@ -181,7 +181,7 @@ def run(self, cmd, log_stdout=True, log_stderr=True,
                                     shell=shell,
                                     cwd=cwd)
             except Exception, e:
-                lgr.error("Failed to start %s: %s" % (cmd, e))
+                lgr.error("Failed to start %r: %s" % (cmd, " under %r" % cwd if cwd else '', e))
```

Comment:
I bet this line is not tested,  since it must fail. 3  arguments but line formatting expects 2